chain/src/txhashset/txhashset.rs

```diff
@@ -260,6 +260,29 @@ impl TxHashSet {
 			.elements_from_insertion_index(start_index, max_count)
 	}
 
+	/// Find a kernel with a given excess. Work backwards from `max_index` to `min_index`
```

Comment:
This might make more sense in the `rewindable_kernel_view` (rather than the `txhashset`).
Maybe a separate PR to refactor this.